avaframe/ana1Tests/rotationTest.py

```diff
@@ -0,0 +1,240 @@
+"""
+Rotation test
+
+This module runs a DFA simulation for different grid alignments
+and compares the results
+"""
+import logging
+import shutil
+from datetime import datetime
+import pathlib
+
+# Local imports
+# import config and init tools
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import fileHandlerUtils as fU
+from avaframe.log2Report import generateReport as gR
+from avaframe.in3Utils import geoTrans as gT
+import avaframe.in2Trans.ascUtils as IOf
+from avaframe.version import getVersion
+# import computation modules
+from avaframe.com1DFA import com1DFA
+# import analysis modules
+from avaframe.ana1Tests import energyLineTest
+# import plotting module
+import avaframe.out3Plot.plotUtils as pU
+# create local logger
+log = logging.getLogger(__name__)
+
+
+def mainRotationTest(avalancheDir, dem, simDF, resTypeList, flagMass, refSimRowHash):
+    """This is the core function of the Rotation Test module
+
+    This module runs the energy line test and rotates the simulation results for the aimec analysis
+
+    Parameters
+    ----------
+    avalancheDir : pathlib path
+        path to avalanche directory
+    dem : dict
+        dem dictionary
+    simDF : dataFrame
+        DFA simulation dataFrame
+    resTypeList : list
+        list of result types available
+    flagMass : boolean
+        should the aimec mass analysis be done
+    refSimRowHash : str
+        row index of the reference simulation in the simDF
+
+    Returns
+    -------
+    simDF : dataFrame
+        DFA simulation dataFrame updated with the energy line test results
+    flagMass : boolean
+        should the aimec mass analysis be done (switched to true if it is an entrainment simulation)
+    """
+    # read energy line config. ToDo should we only read the DFAPath config and remove the energy line one?
+    energyLineTestCfgFile = pathlib.Path('ana1Tests', 'energyLineTest_com1DFACfg.ini')
+    energyLineTestCfg, modInfo = cfgUtils.getModuleConfig(com1DFA, fileOverride=energyLineTestCfgFile, modInfo=True)
+    inDir = avalancheDir / 'Outputs' / 'com1DFA'
+    outDir = avalancheDir / 'Outputs' / 'com1DFARotated'
+    fU.makeADir(outDir)
+    # get reference angle
+    simName = simDF.loc[refSimRowHash, 'simName']
+    relName = (simName.split('_'))[0]
+    thetaRef = float(relName[3:])
+    for simHash in simDF.index:
```

Comment:
I think this (and other occurrences) should be actually named rowSimHash to avoid confusion with the original simHash